src/fuzzing/IsValidSignatureEncoding/libfuzzer-arguments

```diff
@@ -0,0 +1 @@
+-max_len=130
```

Comment:
I think this should be larger.

```
    // Format: 0x30 [total-length] 0x02 [R-length] [R] 0x02 [S-length] [S] [sighash]
    // * total-length: 1-byte length descriptor of everything that follows,
    //   excluding the sighash byte.
    // * R-length: 1-byte length descriptor of the R value that follows.
    // * R: arbitrary-length big-endian encoded R value. It must use the shortest
    //   possible encoding for a positive integer (which means no null bytes at
    //   the start, except a single one when the next byte has its highest bit set).
    // * S-length: 1-byte length descriptor of the S value that follows.
    // * S: arbitrary-length big-endian encoded S value. The same rules apply.
    // * sighash: 1-byte value indicating what data is hashed (not part of the DER
    //   signature)
```

There are multiple 1-byte length fields in the format, so this could miss bugs that rely on long inputs, e.g. if the input needs to max out all of the length fields.

Reply:
Good point. More like 400? Or I guess > 512?